### PR TITLE
Add r-longitudinaldata

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
Needs r-clv

**Comment by dpryan79** [comment]
@bgruening At least historically we've been using selectors with noarch packages, which leads me to suspect that the linter is wrong.

**Comment by dpryan79** [comment]
@conda-forge/help-r This is ready for review. Was there a change in conda such that selectors don't work with noarch packages anymore? There quite a few such R packages that should be updated then.

**Comment by xhochy** [comment]
@dpryan79 No, selectors in `run` weren't working for a long time yet and the bot always correctly reported that. Selectors in `build` and `host` as it is present in many R packages (mostly for `# [win]`) are still working.

**Comment by dpryan79** [comment]
Ah, it’s the runtime selectors that are the issue. Thanks, good to know!

**Comment by dpryan79** [comment]
@conda-forge/help-r Anyone further input here?

**Comment by jdblischak** [comment]
Just for cross-reference, the current behavior of the linter for noarch recipes is to fail if there are selectors present in the `host` or `run` dependencies. This was discussed and implemented in https://github.com/conda-forge/conda-smithy/pull/966

If you wanted to allow selector in the `host` section, [this line](https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/lint_recipe.py#L336) is where the update would need to be made. We decided on the current behavior due to [concerns](https://github.com/conda-forge/conda-smithy/pull/966#issuecomment-450879922) about `run_exports`.

File: recipes/r-longitudinaldata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - r-misc3d
+    - r-rgl
+    - xorg-libxfixes  # [linux]
+  run:
+    - r-base
+    - r-class
+    - r-clv
+    - r-misc3d
+    - r-rgl
+    - xorg-libxfixes  # [linux]
```
Comment (dbast) [inline recipes/r-longitudinaldata/meta.yaml]: Would this succeed if you move it to `test: requires: xorg-libxfixes  # [linux]`? We can try that also after feedstock conversion.

Comment (dpryan79) [reply recipes/r-longitudinaldata/meta.yaml]: Good question, we should try that.

**Pull request merged**